### ISSUE TITLE
Lockscreen widget

### ISSUE THREAD

**Issue by ligi** [ligi/PassAndroid#49]
Add a LockScreen widget. Need to decide first if there is only one pass in there all the time ( perhaps a Switch on the pass ) - or some way to switch between certain passes in the lockscreen. Also what should be displayed there ( only BarCode and title or anything more)

**Comment by Ickleangeleyes** [comment]
I vote for yes a widget to show tickets would be a fantastic edition to the app. An image to help identify (for example if it was a movie an image of the movie), the barcode/qr code,  the date & time would all be helpful to have. I would like switching but it's not a dealbreaker for me.

**Comment by symbolisch1234** [comment]
@ligi i believe in recent versions (>5.0 ?) of android there are no lockscreen widgets anymore. but i would still very much like to see passes on the lockscreen.

the only solution that i know of is checking if passes have a time attached to them, and X minutes before firing up a notification which displays the pass. this would be a great feature (and i believe this is what Passbook on iOS does)

**Comment by ligi** [comment]
Yes - and the issue for this is already open for a long time - should really do this ;-)
